mParticleCore.brs

```diff
@@ -2249,7 +2280,16 @@ function mParticleSGBridge(task as object) as object
             logQoS: function(mediaSession as object, startupTime as integer, droppedFrames as integer, bitRate as integer, fps as integer, options = {} as object) as void
                 m.invokeFunction("media/logQoS", [mediaSession, startupTime, droppedFrames, bitRate, fps, options])
             end function,
+            unixTimeMillis: function() as longinteger
+                date = CreateObject("roDateTime")
+                currentTime = CreateObject("roLongInteger")
+                currentTime.SetLongInt(date.asSeconds())
+                return (currentTime * 1000) + date.getMilliseconds()
```

Comment:
Can you explain to me the logic here? It seems like your getting the timestamp of the current date in ms two ways and then adding it together? Is that right or am I misunderstanding?

Reply:
the reason is because with Brightscript there is no way to get the unix timestamp with milliseconds included, only app to a second, I wanted to match the other media SDKs (and mP in general when dealing with timestamps) to include the unix timestamps including the milliseconds

Reply:
oh so the `(currentTime * 1000)` is like the beginning of the day (like midnight) in ms while date.getMilliseconds() gives you like thats days ms?

Reply:
@mmustafa-tse - you are caling the same code 3x throughout this commit.  can you make a helper function for `getCurrentTimeInMillis` which contains this logic to DRY it up?